.github/workflows/release_infra.yaml

```diff
@@ -7,7 +7,7 @@ on:
       - main
     paths:
       # Trigger the workflow when resources are modified
-      - "infra/resources"
+      - "infra/resources/**"
```

Comment:
should this be backported here https://github.com/pagopa/dx-typescript/blob/main/.github/workflows/release_infra.yml#L10 @Krusty93 @christian-calabrese ?